src/store/validation/index.ts

```diff
@@ -33,12 +34,23 @@ export interface IValidationStatusContext {
   isOrphanResource?: boolean,
 }
 
+/**
+ * Is valid by mode index OR lang index:
+ * - if 1st key is mode index, then the array values are lang index
+ * - if the 1st key is lang index, then the array values are mode index
+ */
+export interface IInvalidResourcesCounterBy {
+  modeIndex?: {[key: number]: number},
+  langIndex?: {[key: number]: number},
+}
```

Comment:
Could you confirm my understanding, please?
An object of `IInvalidResourcesCounterBy` will be shaped as follows:

```js
{
  modeIndex: {
    "0": 12 // there are 12 invalid resources for mode with index "0"
  },
  langIndex: {
    "1": 34 // there are 34 invalid resources for language with index "1"
  }
}
```

If that's the case, we may rename the interface as

```suggestion
/**
 * Number of invalid resources either by mode index or lang index.
 */
export interface IInvalidResourceCounters {
  byModeIndex?: {[key: number]: number},
  byLangIndex?: {[key: number]: number},
}
```

Reply:
you're correct, I will add this example from you to be part of the JS-Doc, thanks ;)

Reply:
and updated the `then the array values are lang index` because that is not true